# right-aligns the absolute numbers wrt to the relative numbers, ocd
diff --git a/src/nvim/drawline.c b/src/nvim/drawline.c
index 08b6fd89a..2fd6a5eca 100644
--- a/src/nvim/drawline.c
+++ b/src/nvim/drawline.c
@@ -402,7 +402,7 @@ static inline void get_line_number_str(win_T *wp, linenr_T lnum, char_u *buf, si
     if (num == 0 && wp->w_p_nu && wp->w_p_rnu) {
       // 'number' + 'relativenumber'
       num = lnum;
-      fmt = "%-*ld ";
+      fmt = "%*ld ";
     }
   }
